# Represents a maliput::api::LaneSRange.
LaneId lane_id
SRange s_range
